src/firefly/js/metaConvert/ShaRequestList.js

```diff
@@ -44,25 +44,68 @@ export function makeShaPlotRequest(table, row, includeSingle) {
     /*
      const url = https://irsa.ipac.caltech.edu/data/SPITZER/SHA/archive/proc/MIPS003600/r5572864/ch2/bcd/SPITZER_M2_5572864_0007_0061_9_bcd.fits
      */
-    let serverinfo = 'https://irsa.ipac.caltech.edu/data/SPITZER/';
+    const serverinfo = 'https://irsa.ipac.caltech.edu/data/SPITZER/';
+    const mosimgserver = 'https://sha.ipac.caltech.edu/applications/Spitzer/SHA/servlet/ProductDownload?DATASET=level1';
```

Comment:
I don't this should be hard coded. In a future iteration it probably needs to be a property.